modules/RTC/BridgeChannel.js

```diff
@@ -170,6 +170,20 @@ export default class BridgeChannel {
     }
 
     /**
+     * Sends a "receiver video constraint" message via the channel.
+     * @param {Number} maxFrameHeightPixels the maximum fram height,
+     * in pixels, this receiver is willing to receive
+     */
+    sendReceiverVideoConstraintMessage(maxFrameHeightPixels) {
+        logger.log('sending a ReceiverVideoConstraint message with '
```

Comment:
the way i see it, there would be a UI element (the quality slider) that would call some logic upon changing which would translate the value of the slider into a constraint